packages/react/src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -220,6 +225,7 @@ const FlyoutMenu = ({
       )}
     >
       <Button
+        {...buttonProps}
```

Comment:
What are we spreading here? Can we not call it out specifically?

Reply:
This was so that the user could also specify the `tooltipPosition` from the default `top`. I can make that prop explicit instead, just thought there might be other use-cases where the button's other props would come into play.